subtests/docker_cli/top/runsleep.py

```diff
@@ -33,8 +33,8 @@ def postprocess(self):
         OutputGood(self.sub_stuff['run_dkrcmd'].cmdresult)
         OutputGood(self.sub_stuff['top_dkrcmd'].cmdresult)
         pstable = TextTable(self.sub_stuff['top_dkrcmd'].stdout)
-        self.failif_ne(len(pstable), 1)
+        self.failif_ne(len(pstable), 1, "Number of rows returned by top")
         psrow = pstable[0]
-        self.failif_ne(psrow['USER'], 'root')
-        self.failif(int(psrow['PID']) == 1)
-        self.failif_ne(psrow['COMMAND'], self.COMMAND)
+        self.failif_ne(psrow['USER'], 'root', 'Expected user')
+        self.failif(int(psrow['PID']) == 1, 'Process PID is 1')
+        self.failif_ne(psrow['COMMAND'], self.COMMAND, 'Expected command')
```

Comment:
Ahh okay.  Don't be afraid to put BZ references into comments.  Yep, that's a very similar / same to the bug I was thinking about.  Let me see if I can't dig it up, may be some helpful comments in it...here it is:  https://bugzilla.redhat.com/show_bug.cgi?id=1096286